Jenkinsfile

```diff
@@ -616,6 +622,54 @@ def withBeatsEnvWin(Closure body){
   }
 }
 
+def dumpFilteredEnivronment(){
```

Comment:
I am not the only one who writes `environment` always in the wrong way 😃 

```suggestion
def dumpFilteredEnvironment(){
```

Reply:
Spelling will be the death of me! Thanks, @kuisathaverat